src/lib/__tests__/vsac_cache.test.ts

```diff
@@ -98,7 +99,7 @@ describe('VsacCache', () => {
     }
   });
 
-  test.skip('should be not load valuesets already cached unless forced', async () => {
+  test('should not load valuesets already cached unless forced', async () => {
```

Comment:
This was commented out because it was not working in tingodb, is it working in mongo?